blueprints/data-solutions/data-platform-foundations/demo/datapipeline.py

```diff
@@ -16,18 +16,15 @@
 # Load The Dependencies
 # --------------------------------------------------------------------------------
 
-import csv
 import datetime
-import io
 import json
-import logging
 import os
+import time
 
 from airflow import models
-from airflow.providers.google.cloud.operators.dataflow import DataflowTemplatedJobStartOperator
+from airflow.providers.google.cloud.operators.dataflow import DataflowStartFlexTemplateOperator
```

Comment:
I would create a separate DAG example, to have 2 examples: running Google provided templates and Flex templates.

I would update the demo README accordingly.

Reply:
Done.